src/components/JoinDiscordBanner/JoinDiscordBanner.style.ts

```diff
@@ -1,18 +1,24 @@
 import type { BoxProps, Breakpoint } from '@mui/material';
-import { Box, Typography, alpha } from '@mui/material';
+import { Typography, alpha } from '@mui/material';
 
 import { urbanist } from '@/fonts/fonts';
 import { styled } from '@mui/material/styles';
+import Link from 'next/link';
 import { IconButtonPrimary } from '../IconButton.style';
 
 export interface DiscordBannerProps extends Omit<BoxProps, 'component'> {
   isArticlePage: boolean;
 }
 
-export const DiscordBanner = styled(Box, {
+export const DiscordBanner = styled(Link, {
```

Comment:
If this is moving from being a Box to a Link might not hurt to rename it to DiscordBannerLink